test/functional/wallet_backwards_compatibility.py

```diff
@@ -351,6 +351,13 @@ def run_test(self):
             descriptor = f"wpkh([{info['hdmasterfingerprint']}{hdkeypath[1:]}]{pubkey})"
             assert_equal(info["desc"], descsum_create(descriptor))
 
+            # Check that descriptor wallets have hd key
+            if self.options.descriptors:
+                descs = wallet.listdescriptors(True)
+                xpub_info = wallet.gethdkey(True)
```

Comment:
in 2657c1295cb156d6021844edf18b1a4dc5ce7136

nit: can also check that HD key is the same as return by the old version (e.g. by checking fingerprint with `hdmasterfingerprint`)

Reply:
Done